inst/schema/StrategiseOptions.schema.json

```diff
@@ -0,0 +1,111 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "budget": {
+      "type": "number"
+    },
+    "zones": {
```

Comment:
I might be misremembering, but I think we decided to standardise on 'region' rather than 'zone'? In which case, it would probably be better to keep that terminology in the schema, though I guess it's not too important since it's not seen by the user (and I think the optimiser code has always dealt in zones). 
UPDATE: I've just read through the rest of the code, and seen it's pretty closely tied in with 'zones', so maybe let's stick with zones here if not in the UI.

Reply:
So the code uses "zone" exclusively. The UI mostly uses "region", and the webapp changes related to strategising are intended to remove the final mentions of "zone" there (even though Tom's latest comments have raised the question of whether region is too linked to geography). But yes, zone here for consistency with remainder of code.